civihr_employee_portal/views/templates/views-view--my-details.tpl.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+  // This view contains 2 other views in its $footer section
```

Comment:
You have added two separate views for phone numbers and email. Is this necessary? why cant we have a single view containing all information and it can be named as `view-mydetails-contact_info`.

Reply:
Because phones and emails are multiple, and you can't print them one row at a time, as views expects. I tried to use only one view but I found issues. I'm afraid that views integration with CiviCRM custom fields is not that good.